api/projects/views.py

```
@@ -110,6 +114,31 @@ def perform_create(self, serializer):  # type: ignore[no-untyped-def]
             UserProjectPermission.objects.create(  # type: ignore[misc]
                 user=self.request.user, project=project, admin=True
             )
+        AuditLog.objects.create(
+            project=project,
+            author=None if is_master_api_key_user else self.request.user,
```

Ruff (0.15.21)

[error] Undefined name `is_master_api_key_user`

(F821)